Server/AuthService/AuthService.Api.Http/AssemblyConfigurator.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.IO;
+using System.Reflection;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+using ModularHouse.Server.Common.Api.Middlewares;
+
+namespace ModularHouse.Server.AuthService.Api.Http;
+
+public static class AssemblyConfigurator
+{
+    public static IServiceCollection ConfigureHttpWebAppServices(this IServiceCollection services)
+    {
+        services.AddControllers();
+        services.AddEndpointsApiExplorer();
+        
+        services.AddSwaggerGen(c =>
+        {
+            c.SwaggerDoc("v1", new OpenApiInfo { Title = "Auth Service API", Version = "v1" });
+            var xmlFile = $"{Assembly.GetExecutingAssembly().GetName().Name}.xml";
+            var xmlPath = Path.Combine(AppContext.BaseDirectory, xmlFile);
+            c.IncludeXmlComments(xmlPath);
+        });
+
+        services.AddExceptionHandlerMiddleware();
+        services.AddTransactionMiddleware();
+        
+        return services;
+    }
+
+    public static WebApplication ConfigureHttpWebApp(this WebApplication app)
+    {
+        if (app.Environment.IsDevelopment())
+        {
+            app.UseSwagger();
+            app.UseSwaggerUI();
+        }
+
+        app.UseHttpsRedirection();
+        app.MapControllers();
+
+        app.UseExceptionHandlerMiddleware();
+        app.UseTransactionMiddleware();
```

Comment:
It's need to call UseTransactionMiddleware before UseExceptionHandlerMiddleware.
If an Exception would throw, then transactionId will set as default
![image](https://github.com/Igasus/modular-house/assets/87234450/1df258c2-e6e8-4db2-a6b5-0c7c97a4c69c)